elf.go

```diff
@@ -45,11 +45,11 @@ type elfFile struct {
 	osFile *os.File
 }
 
-func (e *elfFile) GetParsedFile() any {
+func (e *elfFile) getParsedFile() any {
```

Comment:
This is reverting #74.

Reply:
The public API is still available. Since elf/macho/peFile is not exported, has a public method on it has no meaning.